### PR TITLE
Move .less files into appropriate folders

### PR DESCRIPTION
**Pull request by owennw**

### CODE REVIEW

File: src/assets/styles/style.less [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
 
 @import "variables.less";
-@import "logo.less";
-@import "sidebar.less";
-@import "search.less";
-@import "favourites.less";
-@import "star.less";
-@import "showcase-changes.less";
-@import "custom-scroll.less";
+@import "../../logo.less";
```
Comment (bjedrzejewski) [inline src/assets/styles/style.less]: logo should stay where it is, droping it into main dir is just bad

File: src/assets/styles/style.less [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
+@import "../../sidebars/sidebar.less";
+@import "../../sidebars/search/search.less";
+@import "../../sidebars/favourites/favourites.less";
+@import "../../sidebars/star/star.less";
+@import "../../showcase/showcase-changes.less";
+@import "../../custom-scroll.less";
```
Comment (bjedrzejewski) [inline src/assets/styles/style.less]: Same here, I suggest no main dir as it is messy

File: src/assets/styles/style.less (hunk 1 above)
Comment (owennw) [inline src/assets/styles/style.less]: Yeah I wasn't sure whether it should move or not.. Is that the 'correct' pattern?

File: src/assets/styles/style.less (hunk 2 above)
Comment (owennw) [inline src/assets/styles/style.less]: There is a customScroll controller in the main dir though..

Comment (owennw) [inline src/assets/styles/style.less]: I don't know if that then means it deserves its own folder.. ?

Comment (bjedrzejewski) [inline src/assets/styles/style.less]: I think this means that the .js should be moved rather than the other way round

Comment (bjedrzejewski) [inline src/assets/styles/style.less]: But one issue at a time

File: src/assets/styles/style.less (hunk 1 above)
Comment (bjedrzejewski) [inline src/assets/styles/style.less]: The pattern is- keep the style in assets, unless it directly relates to a directive.

**Pull request merged**